src/sentry/static/sentry/app/components/actions/actionLink.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Confirm from '../confirm';
+import {t} from '../../locale';
+
+export default class ActionLink extends React.Component {
+  static propTypes = {
+    shouldConfirm: PropTypes.bool,
+    message: PropTypes.node,
+    className: PropTypes.any,
+    onAction: PropTypes.func.isRequired,
+    title: PropTypes.string,
+    confirmLabel: PropTypes.string.isRequired,
+    disabled: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    shouldConfirm: false,
+    isDisabled: false,
+  };
+
+  render() {
+    let {
+      shouldConfirm,
+      message,
+      className,
+      title,
+      onAction,
+      confirmLabel,
+      disabled,
+      children,
+    } = this.props;
+
+    let confirmMessage = (
+      <div>
+        {message}
+        <p>{t('This action cannot be undone.')}</p>
```

Comment:
I know that currently this message is incorrectly displayed on some actions that can be undone, maybe we should pass it as part of the message prop instead, per-action?